operatorhub/openshift/release-artifacts/bundle/manifests/openshift-pipelines-operator-rh.clusterserviceversion.yaml

```diff
@@ -28,7 +28,7 @@ metadata:
     description: Red Hat OpenShift Pipelines is a cloud-native CI/CD solution for
       building pipelines using Tekton concepts which run natively on OpenShift and
       Kubernetes.
-    olm.skipRange: '>=1.5.2 <1.6.0'
+    olm.skipRange: '>=1.5.0 <1.6.0'
```

Comment:
trying to understand what does this change does?
previously as it was `>=1.5.2` it was not allowing 1.5.1 to get installed so we are including that in the range now right?

Reply:
this is just an example.. the values in this annotation will depend on how a team want the update graph to look like.

so theoretically, when we make a 1.6.0 release, this annotation will have value `>=1.5.0 <1.6.0`

This annotation will not prevent any version from being installed if it is the latest. The need for this annotation is to state that update can directly jump from minor to minor instead of rolling through patch versions. I state again, this will totally upto the team to decide, depend upon state of update graph and release policy/state of a version.

Reply:
> it was not allowing 1.5.1 to get installed so we are including that in the range now right?

that is unrelated to this.
if we do an annotation like this in our 1.6 release then olm will allow direct upgrades from any `1.5.x` till `1.6.0`. if a user is on 1.5.2 and even though there were othere releases like 1.5.3, 1.5.4, the upgrade from 1.5.2 will directly jump to 1.6.0 for that user.

Reply:
ref: https://olm.operatorframework.io/docs/concepts/olm-architecture/operator-catalog/creating-an-update-graph/#skiprange